generators/app/templates/src/app/components/RadioGroup/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+import RadioOption from './components/RadioOption';
+import { IRadioOption } from './components/RadioOption/interfaces';
+
+interface Props {
+  disabled?: boolean;
+  className?: string;
+  selectedValue?: string;
+  name?: string;
+  options?: any[];
+  onChange?: (event: React.FormEvent<Element>) => void;
+  [rest: string]: any;
```

Comment:
Do you need this attribute?

Reply:
same here, and besides I don't think we want to enable adding more props here